tests/test_cache.py

```diff
@@ -456,19 +423,19 @@ def test_ignore_parameters_post_raw(self):
 
     # TODO: Create mock responses instead of depending on httpbin
     @pytest.mark.skip(reason='httpbin.org/relative-redirect no longer returns redirects')
-    @mock.patch("requests_cache.backends.base.datetime")
-    @mock.patch("requests_cache.core.datetime")
+    @patch("requests_cache.backends.base.datetime")
+    @patch("requests_cache.core.datetime")
     def test_remove_expired_entries(self, datetime_mock, datetime_mock2):
         expire_after = timedelta(minutes=10)
         start_time = datetime.utcnow().replace(year=2010, minute=0)
-        datetime_mock.utcnow.return_value = start_time
+        datetime_utcnow.return_value = start_time
```

Comment:
This should be `datetime_mock` instead of `datetime_utcnow`, or did I miss where that one is declared?

Reply:
I haven't finished going over the unit tests yet, which is why I put the PR in draft mode. I'll probably have those done this weekend.